src/components/containers/ServiceForm/ServiceForm.js

```diff
@@ -10,8 +10,15 @@ import MenuItem from 'material-ui/MenuItem';
 export default class ServiceForm extends Component {
     constructor(props){
         super(props);
-        this.state = {value: 1, verdi: 0};
+        this.state = {value: 1, verdi: 0, showAdvanced: false};
```

Comment:
having a `value` and a `verdi` field isn't very descriptive, and we should avoid mixing languages like that.